src/reporters/CLIReporter.ts

```diff
@@ -25,8 +26,7 @@ export class CLIReporter implements IReporter {
   }
 
   async report(practicesAndComponents: PracticeWithContextForReporter[]): Promise<void> {
-    const reportString = this.buildReport(practicesAndComponents);
-    console.log(reportString);
+    debug.log(this.buildReport(practicesAndComponents));
```

Comment:
`debugLog `?

Reply:
debugLog() doesn't log in terminal. debug.log() logs both in the terminal and in the file if the -l is used.